examples/client/Locomotion/src/context/bottomSheetContext/index.js

```diff
@@ -33,6 +33,8 @@ const BottomSheetProvider = ({ children }) => {
   const [snapPointsState, setSnapPointsState] = useState(SNAP_POINT_STATES[BS_PAGES.LOADING]);
   const [footerComponent, setFooterComponent] = useState(null);
   const [topBarText, setTopBarText] = useState('');
+  const [backgroundColor, setBackgroundColor] = useState(null);
```

Comment:
Im sure there is a better way to display the background color instead of saving it in a context

Reply:
What do you suggest? Background color changes exactly like the text

Reply:
Maybe it would be better to group topBarText, backgroundColor and topBarTextTags together to one object?

Reply:
yes, that's better